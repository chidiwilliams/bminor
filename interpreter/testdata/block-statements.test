a: string = "outer a";
b: string = "outer b";
c: string = "outer c";
{
    b: string = "middle b";
    c: string = "middle c";
    {
        c: string = "inner c";
        print a, "\n", b, "\n", c, "\n";
    }
    print a, "\n", b, "\n", c, "\n";
}
print a, "\n", b, "\n", c;
