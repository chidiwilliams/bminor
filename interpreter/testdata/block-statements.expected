outer a
middle b
inner c
outer a
middle b
middle c
outer a
outer b
outer c
